lib/constants.ts

```diff
@@ -66,7 +73,11 @@ class ItunesConnectApplicationTypesClass implements IiTunesConnectApplicationTyp
 }
 
 export const ItunesConnectApplicationTypes = new ItunesConnectApplicationTypesClass();
-
+export const SYNC_DIR_NAME = "sync";
```

Comment:
Naming the class `LiveSyncPaths` because of a [username taken error](https://github.com/telerik/mobile-cli-lib/blob/master/constants.ts#L19)